gnocchi/gendoc.py

```diff
@@ -99,28 +99,37 @@ def __getitem__(self, key):
 
 
 multiversion_hack = """
+import shutil
 import subprocess
 import sys
 import os
 
+local_branch_path = os.getcwd()
 srcdir = os.path.join("%s", "..", "..")
 os.chdir(srcdir)
 sys.path.insert(0, srcdir)
 
 version = sys.argv[1]
 
-# NOTE(sileht): We delete releasenotes from old documentation
-# only master will have it.
-if os.path.exists("doc/source/releasenotes/index.rst.backup"):
-    os.remove("doc/source/releasenotes/index.rst")
-    os.rename("doc/source/releasenotes/index.rst.backup",
-        "doc/source/releasenotes/index.rst")
-
-if version not in ["<local>", "master"] and os.path.exists("releasenotes"):
-    os.rename("doc/source/releasenotes/index.rst",
-        "doc/source/releasenotes/index.rst.backup")
-    with open("doc/source/releasenotes/index.rst", "w") as f:
-        f.write(\"\"\"
+if version not in ["<local>", "master"]:
+    # NOTE(sileht): Update _static files (mainly logos)
+    if not os.path.exists("doc/source/_static"):
+        os.makedirs("doc/source/_static")
+    for f in ("doc/source/_static/gnocchi-icon.ico",
```

Comment:
is it better if we discover what's in this folder?

Reply:
The idea is to only have the latest version of the logo copied everywhere. But currently if I copy the whole directory it should work.